hack/make/doc/permissions.mk

```diff
@@ -0,0 +1,2 @@
+doc/permissions: manifests
```

Comment:
should we add `python3` to our prerequisites now?

Reply:
If you are referring to the `make prerequisites` step, it might be an overkill. I don't expect that this script is used on a daily basis. Python3 (and pyaml) are needed only by whomever has the honor to update the documentation. These pre-requisites are mentioned in the scripts README. I think that's sufficient.

Reply:
I agree with this being an overkill 👍